src/structures/interfaces/Collector.js

```diff
@@ -37,12 +37,6 @@ class Collector extends EventEmitter {
      */
     Object.defineProperty(this, 'client', { value: client });
 
-    /**
-     * The filter applied to this collector
-     * @type {CollectorFilter}
-     */
-    this.filter = filter;
-
```

Comment:
What's the reasoning behind removing this property? Although it's a very small edge case, wouldn't this prevent someone from changing the filter dynamically?

Could it be kept as `this.filter = options.filter ?? () => true`, removing the need to check for its existence later?

Reply:
Technically, you can do `collector.options.filter = () => true;`, however this will throw if someone freezes their options object and then proceeds to want to alter them. But I'm in favor of bringing this back